examples/webbilling-demo/src/tests/helpers/fixtures.ts

```diff
@@ -40,3 +42,220 @@ export const ITALY_CUSTOMER_DETAILS = {
 export const SPAIN_CUSTOMER_DETAILS = {
   countryCode: "ES",
 };
+
+export const SPAIN_TAX_RESPONSE: RouteFulfillOptions = {
+  status: 200,
+  contentType: "application/json",
+  body: JSON.stringify({
+    mocked: true,
```

Comment:
This parameter is just to confidently be able to spot if the response in playwright was a mocked one or not.